src/kOS/Suffixed/Timespan.cs

```diff
@@ -94,6 +94,7 @@ public double ToUnixStyleTime()
         public static TimeSpan operator -(TimeSpan a, double b) { return new TimeSpan(a.ToUnixStyleTime() - b); }
         public static TimeSpan operator *(TimeSpan a, double b) { return new TimeSpan(a.ToUnixStyleTime() * b); }
         public static TimeSpan operator /(TimeSpan a, double b) { return new TimeSpan(a.ToUnixStyleTime() / b); }
+        public static TimeSpan operator +(double b, TimeSpan a) { return new TimeSpan(b + a.ToUnixStyleTime()); }
```

Comment:
Well good catch then,I suppose.
I sort of wish that operator overloading had a nice syntax that let you specify when order doesn't matter, so you didn't have to explicitly mention it both ways around.  Something like a `commutative` keyword.